todo_app/requests_test_delete.py

```diff
@@ -0,0 +1,10 @@
+import requests
```

Comment:
If this is just for testing and unused (which I think it is) then it's good to not commit it, or remove it after you've done so rather than cluttering the codebase with half-finished experimentation - we can always recover it with Git later
(Similar for the below tests)